test/ussd/fixtures.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function() {
+    return [{
+        "request": {
+            "method": "POST",
+            "url": "http://example.com",
+            "data": {
+                "bar": "baz"
+            }
+        },
+        "response": {
+            "code": 200,
+            "data": {
+                "ham": "spam"
+            }
+        }
+    }];
```

Comment:
These aren't needed yet, but I think it's okay to leave them here for now as long as we remove them as soon as we add the fixtures for the real API calls.